thumbor/engines/pil.py

```diff
@@ -280,6 +292,46 @@ def read(  # noqa
         if options["quality"] is None:
             options["quality"] = self.context.config.QUALITY
 
+        if ext == ".avif":
```

Comment:
I was testing these changes locally, and I didn't install `pillow-avif-plugin` just for test purpose. 

And Thumbor returned error 500 with this log message:

```python
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/thumbor/handlers/__init__.py", line 571, in finish_request
    result = await self.context.thread_pool.queue(
  File "/usr/local/lib/python3.9/dist-packages/thumbor/threadpool.py", line 55, in queue
    return await self._execute_in_pool(operation, *args)
  File "/usr/local/lib/python3.9/dist-packages/thumbor/threadpool.py", line 49, in _execute_in_pool
    return await loop.run_in_executor(self.pool, operation, *args)
  File "/usr/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.9/dist-packages/thumbor/handlers/__init__.py", line 491, in _load_results
    results = context.request.engine.read(image_extension, quality)
  File "/usr/local/lib/python3.9/dist-packages/thumbor/engines/pil.py", line 213, in read
    img_buffer = BytesIO()
  File "/usr/local/lib/python3.9/dist-packages/PIL/Image.py", line 2307, in save
    save_handler = SAVE[format.upper()]
KeyError: 'AVIF'
```

Maybe here, We can validate if `HAVE_AVIF` is True, before accept AVIF extension, or change to default JPEG.

What do you think @fdintino / @heynemann ?

Reply:
oh yes, that's definitely an oversight on my part. I'll add the check and a regression test for this situation.